### PR TITLE
Add/install patches test

### PR DESCRIPTION
**Pull request by feng-j678**
[x] extract vm set up into a method
[x] add install patches testsuit

### CODE REVIEW

**Comment by LiliDeng** [comment]
@feng-j678 please rebase, and resolve the conflict.

File: microsoft/testsuites/vm_extensions/linux_patch_extension.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        Verify status file response for validity.
+        """,
+        priority=2,
+    )
+    def verify_vm_install_patches(
+        self, node: Node, environment: Environment, log: Logger
+    ) -> None:
+        compute_client, resource_group_name, vm_name = _set_up_vm(node, environment)
+        install_patches_input = {
+            "maximumDuration": "PT3H30M",
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/linux_patch_extension.py:115]: will 3 hours 30 mins installation time be conflict with the wait time 600s?

File: microsoft/testsuites/vm_extensions/linux_patch_extension.py [hunk 2]
```diff
@@ -44,8 +44,8 @@ def _set_up_vm(node: Node, environment: Environment) -> Any:
 def _verify_vm_agent_running(node: Node, log: Logger) -> None:
     service = node.tools[Service]
     is_vm_agent_running = service.is_service_running(
-        "walinuxagent"
-    ) or service.is_service_running("waagent")
+        "walinuxagent.service"
+    ) or service.is_service_running("waagent.service")
```
Comment (LiliDeng) [inline microsoft/testsuites/vm_extensions/linux_patch_extension.py]: remove `.service`?

File: microsoft/testsuites/vm_extensions/linux_patch_extension.py [hunk 3]
```diff
@@ -95,3 +94,46 @@ def verify_vm_assess_patches(
...
             "Expected no error in assess patches operation"
         ).is_equal_to("0")
+
+    @TestCaseMetadata(
+        description="""
+        Verify walinuxagent or waagent service is running on vm. Perform install
+        patches to trigger Microsoft.CPlat.Core.LinuxPatchExtension creation in vm.
+        Verify status file response for validity.
+        """,
+        priority=2,
```
Comment (squirrelsc) [inline microsoft/testsuites/vm_extensions/linux_patch_extension.py]: Move it to P3, due to long setup time.

File: microsoft/testsuites/vm_extensions/linux_patch_extension.py (hunk 1 above)
Comment (feng-j678) [reply microsoft/testsuites/vm_extensions/linux_patch_extension.py:115]: spoke to chi on this, he suggested to increase the test timeout count down

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**